src/uu/install/src/install.rs

```diff
@@ -493,7 +494,13 @@ fn copy(from: &PathBuf, to: &PathBuf, b: &Behavior) -> Result<(), ()> {
             );
             return Err(());
         }
-    } else if let Err(err) = fs::copy(from, to) {
+    }
+
+    if b.compare && !need_copy(from, to, b) {
```

Comment:
Please add a comment to explain why it is skipped

Reply:
I found this part (L499) self-explanatory, but I did add documentation to `need_copy`.